### DISCUSSION TITLE
How to hide Computer name and User name in terminal?

### DISCUSSION THREAD

**Discussion by tunv1998** [ohmybash/oh-my-bash#328] [Q&A]
Hi guys, I'm using ohmybash with ```agnoster``` theme, and it display as 

`userName@computername |>path/to/folder`

How can I hide `username` and `computername` and only display `|>path/to/folder`

thanks!

**Comment by nntoan** [comment]
@tunv1998 You can check it here in this article: https://github.com/ohmybash/oh-my-bash/wiki/Customization and also take a note of this: https://github.com/ohmybash/oh-my-bash#customization-of--plugins-and-themes

**Reply by anish137i** [reply]
what is we use random theme is there any solution available to hide Hostname and username the default method in .bashrc PS1 not work ?
